packages/commonwealth/server/routes/status.ts

```diff
@@ -117,7 +117,7 @@ export const getUserStatus = async (models: DB, user: UserInstance) => {
       selectedCommunity,
       isAdmin,
       disableRichText,
-      communities: userCommunities?.[0] || [],
+      starredCommunities: userCommunities?.[0] || [],
```

Comment:
I think the type should be updated not this return statement. Check the client. The user store expects `communities` not `starredCommunities`.

Reply:
let me put just both on the server for now.

Reply:
Why though? We can see the type is wrong in the API. https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/scripts/controllers/app/login.ts#L247

Reply:
Removing `starredCommunities` from the type is the correct path forward since it resolves a bug in #9021 and is the correct solution according to #8877:
> Updated /status API, to return user joined communities list with is_starred boolean, deprecating the starredCommunities response and state from the status route and frontend.

Reply:
Works for me. Going to nuke it !!